src/components/PositionMarker.js

```diff
@@ -11,9 +11,7 @@ const PositionMarker = ({ location, setLocation, markerLocation, setMarkerLocati
 
   if (location && typeof location === "string") {
     const splitPosition = location.split(",");
-    const lat = splitPosition[0];
-    const lon = splitPosition[1];
-    location = [lat, lon];
+    location = [splitPosition[0], splitPosition[1]];
```

Comment:
I thought that t`ypeof location === "string"` looked strange, since you are setting it to an array.
But then I saw that location is sent down inside of  LeafletMap
![image](https://user-images.githubusercontent.com/44663708/156170108-3aed287e-92ef-41c7-899f-9762dce5daf2.png)
But location isnt sent down to LeafletMap
![image](https://user-images.githubusercontent.com/44663708/156170193-f746788d-6247-472d-901e-45645e05e61c.png)

So Im a bit confused...

Reply:
location is sent down to AddMap in Add and Edit. But the component AddMap should probably be named AddEditMap.